apps/cowswap-frontend/src/cosmos.decorator.tsx

```diff
@@ -19,11 +19,9 @@ import store from 'legacy/state'
 import { useDarkModeManager } from 'legacy/state/user/hooks'
 import ThemeProvider, { FixedGlobalStyle, ThemedGlobalStyle } from 'legacy/theme'
 
-import { injectedConnection } from 'modules/wallet/web3-react/connection/injected'
-
 import { BlockNumberProvider } from 'lib/hooks/useBlockNumber'
 
-import { WalletUpdater } from './modules/wallet'
+import { WalletUpdater, injectedConnection } from './modules/wallet'
```

Comment:
why cosmos don't use absolute imports?

Reply:
There was a circular dependency that broke cosmos build :( Probably after [this changes](https://github.com/cowprotocol/cowswap/pull/3109) it won't be relevant